services/reddit/subreddit-subscribers.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const { BaseJsonService, NotFound } = require('..')
+const Joi = require('joi')
+const { optionalNonNegativeInteger } = require('../validators')
+const { metric } = require('../text-formatters')
+
+const schema = Joi.object({
+  data: Joi.object({
+    subscribers: optionalNonNegativeInteger,
+  }).required(),
+}).required()
+
+module.exports = class SubredditSubscribers extends BaseJsonService {
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'reddit/subreddit-subscribers',
+      pattern: ':subreddit',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Subreddit subscribers',
+        namedParams: { subreddit: 'drums' },
+        staticPreview: {
+          label: 'follow r/drums',
+          message: '77k',
+          color: 'red',
+          style: 'social',
+        },
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'reddit',
+    }
+  }
+
+  static render({ subreddit, subscribers }) {
+    return {
+      label: `follow r/${subreddit}`,
+      message: metric(subscribers),
+      color: 'red',
+      link: [`https://www.reddit.com/r/${subreddit}`],
```

Comment:
is there a link that can be used for users to perform the action (i.e. to actually subscribe)?

Reply:
Hi @calebcartwright. I've been researching this, and I think it's not possible. There's no available link to subscribe to the subreddit. 😔

Reply:
No worries, thanks for looking into it!